src/DynamoCoreWpf/Views/Core/AnnotationView.xaml.cs

```diff
@@ -107,6 +107,21 @@ private void AnnotationView_OnMouseLeftButtonDown(object sender, MouseButtonEven
             {
                 DynamoSelection.Instance.ClearSelection();
             }
+
+            //When the Zoom * Fontsized factor is less than 7, then
```

Comment:
Never drop numbers like this in the comments.  They'll easily become inaccurate.